src/index.html

```diff
@@ -8,7 +8,205 @@
+                  <img class="social-media-btn" src="" alt="linkedin" />
+                  <img class="social-media-btn" src="" alt="twitter" />
+                  <img class="social-media-btn" src="" alt="facebook" />
+                </div>
+                <div id="personal-info-aboutMe-left-p">
+                  <p>
+                    I'm a paragraph. Click here to add your own text and edit
+                    me. It’s easy. Just click “Edit Text” or double click me to
+                    add your own content and make changes to the font. Feel free
+                    to drag and drop me anywhere you like on your page. I’m a
+                    great place for you to tell a story and let your users know
+                    a little more about you.
+                  </p>
+                </div>
+                <div id="personal-info-aboutMe-right-p">
+                  <p>
+                    I'm a paragraph. Click here to add your own text and edit
+                    me. It’s easy. Just click “Edit Text” or double click me to
+                    add your own content and make changes to the font. Feel free
+                    to drag and drop me anywhere you like on your page. I’m a
+                    great place for you to tell a story and let your users know
+                    a little more about you.
+                  </p>
+                </div>
+              </li>
+
+              <li class="carousel-cards" id="personal-info-hobbies">
+                <h2 class="personal-info-cards-title">HOBBIES & INTERESTS</h2>
+                <ul class="personal-info-hobbies-list">
+                  <li class="personal-info-hobbies-item">
+                    <img
+                      class="personal-info-hobbies-item-icon"
+                      src=""
+                      alt="cinema-icon"
+                    />
+                    <span class="personal-info-hobbies-item-label">Cinema</span>
+                  </li>
+                  <li class="personal-info-hobbies-item">
+                    <img
+                      class="personal-info-hobbies-item-icon"
+                      src=""
+                      alt="sport-icon"
+                    />
+                    <span class="personal-info-hobbies-item-label">Sport</span>
+                  </li>
+                </ul>
+              </li>
+
+              <li class="carousel-cards" id="personal-info-eduexp">
+                <h2 class="personal-info-cards-title">Education</h2>
+                <div class="personal-info-eduexp-cards-group">
+                  <div class="personal-info-eduexp-card">
+                    <h3 class="personal-info-eduexp-card-name">
+                      The San Francisco Business School
+                    </h3>
+                    <span class="personal-info-eduexp-card-period"
+                      >2014 - Current</span
+                    >
+                    <span class="personal-info-eduexp-card-desc"
```

Comment:
Use a block tag for beginning with a new line. So `p` or `div` will be more suitable.